.github/workflows/test.yml

```diff
@@ -220,6 +228,86 @@ jobs:
             echo "=> go test CGO_ENABLED=1 w/o optimization"
             env CGO_ENABLED=1 go test "-gcflags=all=-N -l" -v ./...
 
+            if [ -z "$(echo ${{matrix.go}} | grep '^1.1')" ]; then
+              echo "=> go test race"
+              go test -race -shuffle=on -v -count=10 ./...
+            fi
+  netbsd:
+    strategy:
+      matrix:
+        go: ['1.21.13', '1.22.12', '1.23.6', '1.24.0']
```

Comment:
1.23.7 and 1.24.1?

Reply:
I was gonna do that after this got merged since FreeBSD also need to be updated but if you don't care if I do both in this PR I can do it now

Reply:
Acknowledged. Let's do this in the next PR.